new tailwindcss and alpinejs front-end

stesla opened the pull request:
I produced `public/market.json` by scraping the data on the current site, so that I could get the front-end going before we got the back-end going.

lizthegrey commented on package.json:
```diff
@@ -0,0 +1,25 @@
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/stesla/market-site.git"
```
use eve-val instead through here

lizthegrey commented on public/market.json:
```diff
@@ -0,0 +1,2866 @@
+[
```
don't check this file in; this will be autogenerated by the python code in the parent dir

stesla commented:
Removed market.json from history and fixed the URL. Also incorporated some UI feedback I got from one my team's designer after showing off screenshots.

lizthegrey commented on package.json:
```diff
@@ -0,0 +1,25 @@
+  },
+  "author": "Sam Tesla <samuel.tesla@gmail.com>",
+  "license": "ISC",
+  "bugs": {
```
Same problem here with the URLs

stesla replied:
Hrm. What should we put there?

stesla replied:
I suppose we can probably omit author, tbh. But do we want something for license?

lizthegrey commented:
https://github.com/eve-val/market-site/blob/master/LICENSE already MIT license

stesla commented:
Of course it is. I'm pretty sure my eyes just glazed over it when I was filling out package.json. Fixed it in the first commit so as not to have it wrong at any point.

lizthegrey commented on package.json:
```diff
@@ -0,0 +1,25 @@
+  },
+  "author": "Sam Tesla <samuel.tesla@gmail.com>",
+  "license": "MIT",
+  "bugs":
```
invalid json here, needs a `{`?

stesla commented:
Okay, I added a `--json` flag that works just like the `--text` flag and outputs to stdout. Tested the result with the front-end, works like a charm.

lizthegrey commented:
> Okay, I added a `--json` flag that works just like the `--text` flag and outputs to stdout. Tested the result with the front-end, works like a charm.

Almost there! One last change request: change the shell script to create both LSC4-P.html and market.json in the public dir and then cp them (and the css/js files) over to /srv/
